src/test/java/wooteco/subway/service/line/LineServiceTest.java

```diff
@@ -22,10 +22,10 @@
 
 @ExtendWith(MockitoExtension.class)
 public class LineServiceTest {
-    private static final String STATION_NAME1 = "강남역";
-    private static final String STATION_NAME2 = "역삼역";
-    private static final String STATION_NAME3 = "선릉역";
-    private static final String STATION_NAME4 = "삼성역";
+    private static final String STATION_NAME_KANGNAM = "강남역";
```

Comment:
```suggestion
    private static final String 강남역 = "강남역";
```
변수명 자체를 한글로 쓸 수도 있어요
한글로 쓰게 되면 어떤 변수인지 좀더 파악하기 쉽죠